### ISSUE TITLE
[Feature Request] No recent pip wheel available for linux arm64 

### ISSUE THREAD

**Issue by ttim** [pytorch/tensordict#1138]
## Motivation

We use TensorDict in our code and trying to run it on GH200 nodes, only old version is available (0.2.x I think), while recent versions are available only for x64.

## Solution

It would be great to have prebuilt binaries available for linux arm64 systems.

## Alternatives

Either built from source (I guess?), or not use it 

## Additional context

GH200 system

## Checklist

- [x] I have checked that there is no similar issue in the repo (**required**)

**Comment by vmoens** [comment]
Yeah you're right. iirc I didn't manage to upload them on pypi so I had to put them in the release notes

https://github.com/pytorch/tensordict/releases/tag/v0.6.2
